automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoModelCopy.scala

```diff
@@ -66,7 +155,7 @@ case class Cluster(clusterName: ClusterName,
 
 case class ClusterRequest(bucketPath: GcsBucketName,
                           labels: LabelMap,
-                          jupyterExtensionUri: Option[GcsPath] = None)
+                          jupyterExtensionUri: Option[String] = None)
```

Comment:
Why this change? This allows us to immediately barf at the user when interpreting the JSON if someone gives us something that's not shaped like a bucket path.

Reply:
I was running into issues with deserializing the GcsPath, for the tests  its easier to send this request parameter as a String instead of a GcsPath

Reply:
One day we'll publish leo-model and we'll be straight back to this issue, though.

Reply:
Current Leo Swat has lots of this.  The only reason this wasn't covered before is because I only had `None` for these in my tests.

Reply:
fair. we'll have to fix this At Some Point but we can wait until then